packages/radix/src/components/Button.tsx

```diff
@@ -11,11 +11,13 @@ const waitingAnimation = (props: ButtonProps) => keyframes`
 	}
 `;
 
-type Variants = 'gray' | 'blue' | 'green' | 'red' | 'active' | 'waiting';
+type Variants = 'gray' | 'blue' | 'green' | 'red';
```

Comment:
what was the reasoning behind moving to some boolean props? I thought we were trying to avoid these?

Reply:
Yeah, we were. But it doesnt make sense to group all of those into one variant I think.

Maybe we could group `active` and `waiting` into one? Perhaps something like: `mode`?

Otherwise I'm ok with using boolean props as long as they make sense...

Reply:
Well I suppose it depends, for me if you start to make 2 different groups, I would then expect say the `waiting` one to be themed inline with the `variant` color. See what I mean? I feel like that's not the case right now, so really they are all mutually exclusive right now, which is why I feel all of them being in the same `variant` made sense.